controller/raft/errors.go

```diff
@@ -0,0 +1,25 @@
+package raft
+
+import (
+	"github.com/pkg/errors"
+	"reflect"
+)
+
+type Error interface {
```

Comment:
Doc here please. Seeing an interface type called `Error` throws up so many questions.

Reply:
Removing for now. Wasn't actually used anymore.